content/python/encyclopedia/classes-in-python.md

```diff
@@ -0,0 +1,47 @@
+<!--
+---
+Subjects:
+  - "What are Classes in Python?"
+Tags: 
```

Comment:
Please use tags listed here: https://author.codecademy.com/browse/tags
"python" is on that list, but the other two are not